Userland/Utilities/profile.cpp

```diff
@@ -5,11 +5,13 @@
  */
 
 #include <LibCore/ArgsParser.h>
+#include <LibMain/Main.h>
+#include <LibCore/System.h>
```

Comment:
Includes should be sorted alphabetically. (This is what the CI is complaining about.) Running `clang-format` will do that for you, and you can configure most editors to run it automatically.